pylib/spinnaker/yaml_util.py

```diff
@@ -16,6 +16,27 @@
 import re
 import yaml
 
+def yml_or_yaml_path(basedir, basename):
+  """Return a path to the requested YAML file.
+
+  Args:
+    basedir [string]: The directory path containing the file.
+    basename [string]: The base filename for the file
+
+  Returns:
+    Path to the YAML file with either a .yml or .yaml extension,
+    depending on which if any exists.
+    If neither exists, return the .yml. If both, then raise an exception.
```

Comment:
what do you mean? I thought that's what I do.

Oh, it doesnt matter if it exists or not since the default path with be the .yml whether or not it exists. It only matters if it exists if the .yaml exists as a check for an ambiguous file which technically might be permitted by the spring boot library but I'm flagging it as an error since it doesnt seem necessary to permit and more likely suggests a typo or future maintainence problem.